mlflow/utils/validation.py

```diff
@@ -639,3 +640,16 @@ def _validate_experiment_artifact_location_length(artifact_location: str):
             "MLFLOW_ARTIFACT_LOCATION_MAX_LENGTH environment variable.",
             INVALID_PARAMETER_VALUE,
         )
+
+
+def _validate_logged_model_name(name: Optional[str]) -> None:
+    if name is None:
+        return
+
+    disallowed_chars = ("/", ":", ".")
+    if not name or any(c in name for c in disallowed_chars):
```

Comment:
This aligns with Databricks workspace registry.

Reply:
I tried to use the same validation rule as registered model name, but it allows `/` or `:`.